sfdx-source/apex-common/test/classes/fflib_SObjectUnitOfWorkTest.cls

```diff
@@ -848,6 +905,43 @@ private with sharing class fflib_SObjectUnitOfWorkTest
         }
     }
 
+    private class MockDMLWithInsertIds implements fflib_SObjectUnitOfWork.IDML
```

Comment:
@ClayChipps and @jprichter -- Any reason why you did not just simply modify the existing `MockDML`??

Reply:
Updated to use the existing MockDML.